metabase/metabase
# Add pinned card loading states

Epic https://github.com/metabase/metabase/issues/23222

Adds animated loading states to solve some issues with slow pinned questions.

How to test:
- Modify https://github.com/metabase/metabase/blob/d0165b493b3bae80068e3cb361609b41ffa5af31/frontend/src/metabase/collections/components/PinnedChartCard/PinnedChartCard.tsx#L47 to `true` instead of `loading`
- Create questions with all visualization types and pin them
- Open the collection with these questions
- You should see the corresponding loading image
- Make sure dnd and the item menu works

<img width="514" alt="Screenshot 2022-06-10 at 18 00 22" src="https://user-images.githubusercontent.com/8542534/173094417-25c0e98c-aa83-4669-9514-52b9031bbd55.png">
